### DISCUSSION TITLE
Go to last menu choice with double-push on left button

### DISCUSSION THREAD

**Discussion by PsyK0p4T** [sanni/cartreader#559] [Ideas & Suggestions]
With the recent updates, behaviors changed a little bit on screen (I'm using hw3).

Pushing left button once : still goes to next option
Pushing left button twice fast : still goes to previous option
Pushing left button twice fast when you are on the first option : doesn't go to last option anymore.

I loved this possibility, to quickly go on "Reset" option which is often at bottom. Any chance to get this back ?

**Comment by sanni** [comment]
Fixed. It was a bug in the HW5 code that I never noticed and now that I moved HW3 to use the HW5 code it got to HW3 too.

**Reply by PsyK0p4T** [reply]
Tested with latest commit, everything's perfect again ! Thanks a lot :)
